js/scripts/util.js

```diff
@@ -271,7 +281,48 @@ const EVENT_HANDLERS = {
         const scaleY = CANVAS.height / rect.height;
         GAME.mousePos = new Vector((mouse.clientX - rect.left) * scaleX, (mouse.clientY - rect.top) * scaleY);
     },
+
     gameplayKeyDown: (key) => {
+
+        if (key.code === "KeyM" && mickeyCount < 1 && keyPressCounter.M_KEY < keyPressCountMax) {
+            mickeyCount++;
+            keyPressCounter.M_KEY++;
+
+
+        } else if (key.code === "KeyI" && mickeyCount < 2 && keyPressCounter.I_KEY < keyPressCountMax) {
+            mickeyCount++;
+            keyPressCounter.I_KEY++;
+
+        } else if (key.code === "KeyC" && mickeyCount < 3 && keyPressCounter.C_KEY < keyPressCountMax) {
+            mickeyCount++;
+            keyPressCounter.C_KEY++;
+
+        } else if (key.code === "KeyK" && mickeyCount < 4 && keyPressCounter.K_KEY < keyPressCountMax) {
+            mickeyCount++;
+            keyPressCounter.K_KEY++;
+
+        } else if (key.code === "KeyE" && mickeyCount < 5 && keyPressCounter.E_KEY < keyPressCountMax) {
+            mickeyCount++;
+            keyPressCounter.E_KEY++;
+
+        } else if (key.code === "KeyY" && mickeyCount < 6 && keyPressCounter.Y_KEY < keyPressCountMax) {
+            mickeyCount++;
+            keyPressCounter.Y_KEY++;
+
+        } else {
+            mickeyCount = 0;
+            keyPressCounter.M_KEY = 0;
+            keyPressCounter.I_KEY = 0;
+            keyPressCounter.C_KEY = 0;
+            keyPressCounter.K_KEY = 0;
+            keyPressCounter.E_KEY = 0;
+            keyPressCounter.Y_KEY = 0;
+
+        }
+        console.log(mickeyCount);
+        if (mickeyCount == 6) {
+            window.location.href = "https://www.youtube.com/watch?v=hmzO--ox7X0";
```

Comment:
This is fun. Will it cause the game progress to be lost.. unsaved?

Reply:
Thanks for pointing this out! We can either do a save before the link used or open in a new tab which is likely possible. @B1GD3VD0G how difficult would it be to just add another save at this point?

Reply:
For now I made it open a new page.